### PR TITLE
Bump RBC to 0.9.1

### PR DESCRIPTION
**Pull request by guilhermeleobas**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,8 +8,8 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.org/packages/source/{{ pypi_name[0] }}/{{ pypi_name }}/{{ pypi_name }}-{{ version }}.{{ bundle }}
-  sha256: 27ef00d1938e8186153867fb6f10ea2e5631db95e30677ecd9a8a556562e51b7
+  url: https://github.com/xnd-project/rbc/archive/refs/tags/v{version}.tar.gz
```
Comment (tupui) [inline recipe/meta.yaml:11]: Why the change from PyPi?

Comment (guilhermeleobas) [reply recipe/meta.yaml:11]: I couldn't download the source to check the sha256

Comment (tupui) [reply recipe/meta.yaml]: Works on my side:

```
wget https://pypi.org/packages/source/r/rbc-project/rbc-project-0.9.1.tar.gz
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -8,8 +8,8 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.org/packages/source/{{ pypi_name[0] }}/{{ pypi_name }}/{{ pypi_name }}-{{ version }}.{{ bundle }}
-  sha256: 27ef00d1938e8186153867fb6f10ea2e5631db95e30677ecd9a8a556562e51b7
+  url: https://github.com/xnd-project/rbc/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: beceee52c18fd55ae2e95f62859034e41558b613a8372b0f45b80c054a26700a
```
Comment (tupui) [inline recipe/meta.yaml:12]:
```suggestion
  sha256: a9d3acbd691f354a22e4eb3f8ec3349607cc7975c87ae06246f9aa20fecebc32
```
